people-and-planet-ai/image-classification/noxfile_config.py

```diff
@@ -23,7 +23,7 @@
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
     # NOTE: Apache Beam does not currently support Python 3.9.
```

Comment:
I'm assuming 3.10 is also not supported, should we add this to the `NOTE:` in line 25?

Reply:
Correct, Beam only supports up to 3.8, so 3.9 and 3.10 are not supported by Beam. I'll update the note.